docs/rules/space-infix-ops.md

```diff
@@ -20,23 +23,49 @@ While this is valid JavaScript syntax, it is hard to determine what the author i
 
 This rule is aimed at ensuring there are spaces around infix operators.
 
-## Options
+Examples of **incorrect** code for this rule:
 
-This rule accepts a single options argument with the following defaults:
+```js
+/*eslint space-infix-ops: "error"*/
 
-```json
-"space-infix-ops": ["error", {"int32Hint": false}]
+const quotient = dividend/divisor;
+const cubed = x**3;
 ```
 
-### `int32Hint`
-
-Set the `int32Hint` option to `true` (default is `false`) to allow write `a|0` without space.
+Examples of **correct** code for this rule:
 
 ```js
-var foo = bar|0; // `foo` is forced to be signed 32 bit integer
+/*eslint space-infix-ops: "error"*/
+
+const quotient = dividend / divisor;
+const cubed = x ** 3;
 ```
+## Options
+
+* `all` set to `always` requires spaces around all operators that do not override this (the default).
+* `all` set to `never` disallows spaces around all operators that do not override this.
+* `all` set to `ignore` doesn't enforce any particular use of spaces around operators that do not override this.
+* The name of every operator (except `in` and `instanceof`)
+  can be used as an option with the values
+  `always` (require spaces), `never` (disallow spaces), or `ignore` (don't care).
+  These options override the `all` setting for that operator.
+  Options for `in` and `instanceof` are not supported because
```

Comment:
Specifying `in : ignore` will have no effect.
This line makes is so regardless of the options specified, there must be spaces around the `in` and `instanceof` operators.
```js
        const spacesYes = { in: true, instanceof: true };
```